lib/fetch/sanitize.spec.js

```diff
@@ -91,6 +96,8 @@ describe('data/sanitize', () => {
         datetime: null,
         startDateTime: null,
         endDateTime: null,
+        buttonLink: 'exampleLink',
+        status: 'WAITLIST',
```

Comment:
That's not a null check test, that's just extracting the value. The sanitize function does not assign the default value, that is done in the event schema resolve function